flake.nix

```diff
@@ -0,0 +1,55 @@
+{
+  description = "Python package for controlling an ElGato key light";
+
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs";
+
+  outputs = { self, nixpkgs }:
+  let
+    pkgs = nixpkgs.legacyPackages.x86_64-linux;
+
+    leglight = pkgs.python38Packages.buildPythonPackage rec {
+      pname = "leglight";
+      version = "0.2.0";
+
+      src = pkgs.python38Packages.fetchPypi {
+        inherit pname version;
+        sha256 = "41ab462fe12e2ec3e02ff29a00316f822b078c331c70bd36a635568bd1c4204a";
+      };
+
+      propagatedBuildInputs = with pkgs.python38Packages; [
+        zeroconf
+        requests
+      ];
+
+      doCheck = false;
+    };
+  in {
+    devShell.x86_64-linux = pkgs.mkShell {
+      buildInputs = with pkgs; [
+        python38
+        pipenv
+      ];
+    };
+
+    defaultPackage.x86_64-linux = pkgs.python38Packages.buildPythonPackage rec {
```

Comment:
Does this package only support x86_64-linux? Theres a project called flake-utils you can check out which makes this process easier, so you can go `utils.lib.eachDefaultSystem (system: defaultPackage = ...; )` and the outputs get "system spaced".